### PR TITLE
Add status filter and update readme to include getIncident

### PR DESCRIPTION
**Pull request by mccarthy**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ npm install pagerduty
     * `description` - Text that will appear in the incident’s log associated with this event.
     * `callback` - A Callback with two arguments `(err, response)`
 
+### getIncident(options)
+  * Required
+    *`incidentKey` - Incident key to retrieve incidents for.
```
Comment (svolpe43) [inline Readme.md]: Missing a space after \* causing it to not return.

**Comment by svolpe43** [comment]
Are these changes something to  merge with https://github.com/majorleaguesoccer/node-pagerduty

File: Readme.md (hunk 1 above)
Comment (mccarthy) [inline Readme.md]: Fixed.

**Comment by mccarthy** [comment]
I actually wasn't sure what to do on that front.  Lots of forks, the original is:
https://github.com/skomski/node-pagerduty

I can create a PR with the MLS one and see what happens one this is merged.

**Comment by mccarthy** [comment]
As discussed, I now think this fork was unnecessary, as calling resolve when there's no incident is a no-op.  

https://developer.pagerduty.com/documentation/integration/events/resolve

**Pull request merged**